docs/_articles/ja/user-guide/perf-enhancements.md

```diff
@@ -1,24 +1,39 @@
+The Apex, Deploy and Retrieve commands are in the process of moving to a new architecture. These commands have been rearchitected into independent Typescript libraries and CLI plugins as part of our larger effort to break up the salesforce-alm plugin.  [You can read more about that in this blog post](https://developer.salesforce.com/blogs/2021/02/open-sourcing-salesforce-cli-update-feb-2021.html). There’s many great benefits to this effort and one of the best is better performance. For single file deploys and retrieves, you'll likely notice a significant performance gain when using the latest architecture.
 
-> 注意: コードパフォーマンスを改善させる変更は現在ベータ版です。バグを発見したかフィードバックがある場合は、[GitHub に issue をオープン](./ja/bugs-and-feedback)してください。
+## Execution Process and Status
 
-## 設定
+This architecture change is done through a phased approach.  For both the Apex library and Deploy/Retrieve library, we are rolling out the changes through these phases:
 
-このベータ機能を有効化するには、以下の手順に従います。
+1. Library created and completed.
+2. Available to try it in VS Code - We provide a VS Code setting so you can opt in and try it out. 
+3. On by default in VS Code - The VS code setting default is defaulted to True to enable the library. If you previously opted in and then opted out, you'll have to manually opt back in to use the feature.  We'll change the default after the commands have been available and used for months.
+4. CLI updated to use the library.  The VS Code setting is then retired.  Both the CLI and VS Code extensions are using the library.
 
-1. **[File \(ファイル\)]** > **[Preferences \(基本設定\)]** > **[Settings \(設定\)]** (Windows or Linux) または **[Code \(コード\)]** > **[Preferences \(基本設定\)]** > **[Settings \(設定\)]** を選択します。
-2. Salesforce Feature Previews にある、Experimental: Deploy Retrieve を選択します。
 
-このベータ版リリースでは、以下のメタデータ型に対して **SFDX: Deploy This Source to Org (SFDX: このソースを組織にデプロイ)** または **SFDX: Retrieve This Source From Org (SFDX: このソースを組織から取得)** を実行した際に、パフォーマンス改善が有効になります。
 
-- Apex クラス
-- Apex トリガ
-- Visualforce コンポーネント
-- Visualforce ページ
-- Lightning コンポーネント
-- Aura Components
+
+
+|                         | Library complete	| Available to try in VS Code	| On by default in VS Code	| CLI Updated  |
+|-------------------------|:-----------------:|:---------------------------:|:-------------------------:|:------------:|
+| Apex Library	          |        ✔️          |               ✔️             |             ✔️             |       ✔️        |
+| Deploy Retrieve Library |        ✔️          |               ✔️             |             ✔️              |              |
+	
+
+## Setup
+
+There is a VS Code setting for each library.  Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).  Setting to false means that VS Code will call the CLI to execute those commands. 
```

Comment:
Make the same change here as above? Each library has a corresponding VS Code setting. Access them by selecting Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS). Setting to false means that VS Code uses the CLI to execute those commands.

Reply:
I went ahead and suggested below to make it easier for you.

Reply:
Thanks Emily! I added in the edit.